packages/docusaurus-theme-search-algolia/package.json

```diff
@@ -25,7 +25,7 @@
     "build:format": "prettier --config ../../.prettierrc --write \"lib/**/*.js\""
   },
   "dependencies": {
-    "@docsearch/react": "^3.0.0-alpha.50",
+    "@docsearch/react": "^3.0.0",
```

Comment:
@shortcuts Is this safe?

Reply:
It is safe, but introduces a breaking change in DocSearch.

We now require users to provide an `appId`, which is a follow up of [migrating to our new infra](https://docsearch.algolia.com/docs/migrating-from-legacy). All of the users have received their new credentials but not all of them migrated yet.

I believe upgrading will encourage people to migrate.

Reply:
Yes, I was asking on Discord as well, but thought it may be faster to ask here :P

This means it will break for a minority of our users, but they could and should have upgraded right?

Reply:
> Yes, I was asking on Discord as well, but thought it may be faster to ask here :P

Sorry, I'll paste my answer to give users visibility on this too

> This means it will break for a minority of our users, but they could and should have upgraded right?

Yes indeed, we are no longer maintaining the legacy infrastructure and are definitely pushing to the new infra, so this upgrade will also help!

Reply:
Yes, we already agreed on Discord to do this upgrade

Maybe it could be better to do it in a separate PR to make it clear there's a breaking change @Josh-Cena ? 

We also need to update the validation to mark appId as required + link to the PR or blog post so that we can explain this better to users

Reply:
Uh, going to do it in this PR. This is the only significant bump anyways, otherwise I won't even put this in the changelog (I'm recently excluding some maintenance PRs from the changelog)